generators/diffractivePhaseSpace.cc

```diff
@@ -152,37 +152,33 @@ diffractivePhaseSpace::event()
 
 	bool done = false;
 	do {
-		double tPrime;
-		double xMass;
 		do {
-			if(not (*_pickerFunction)(xMass, tPrime)) {
+			if(not (*_pickerFunction)(_massX, _tPrime)) {
```

Comment:
why `_massX` and not `_xMass`? There are lot of occurrences of `xMass` (e.g. in `xMassMax`, `xMass2`, maxXMassSlices`, ...) still remaining in the following.